src/lTerm_widget.mli

```diff
@@ -253,6 +256,31 @@ class ['a] radiobutton : 'a radiogroup -> string -> 'a -> object
 
 end
 
+(** {6 Scrollbars} *)
+
+(** methods required for a widget to supprt scrollbars *)
+class type scrollable = object
+  method full_size : LTerm_geom.size
+    (* full size of scrollable window *)
+
+  method offset : LTerm_geom.coord
```

Comment:
Do you mean something like add a `#adjustment` interface to the scrollbars with some API functions for getting/setting parameters, then passing the scrollbars to the scrollable/adjustable widget (ie via a constructor or somesuch)?